modules/core/src/persistence_impl.hpp

```diff
@@ -0,0 +1,228 @@
+#ifndef OPENCV_CORE_PERSISTENCE_IMPL_HPP
+#define OPENCV_CORE_PERSISTENCE_IMPL_HPP
+
+#include "precomp.hpp"
```

Comment:
> precomp.hpp

must be used in .cpp files only (as unconditional first include)

Reply:
Fixed